README.md

```diff
@@ -2,3 +2,17 @@
 Welcome to pyensign!
 
 This repository contains the Ensign driver, SDK, and helpers for Python. For the main ensign repo, go [here](https://github.com/rotationalio/ensign). We also have SDKs for [Javascript](https://github.com/rotationalio/ensignjs) and [Go](https://github.com/rotationalio/goensign).
+
+## Building the protocol buffers
+
+This repo relies on [protocol buffers](https://protobuf.dev/) for code generation. If you need to rebuild the protocol buffers, clone the ensign repo to the parent directory.
```

Comment:
Thank you for including this! I'm guessing that in the future we might need to add some more details about how to build protoc since I bet this'll be new for a lot of Python folks, but we can use this as a jumping off point!